Add CS Engine package links and offline install instructions

mstanleyjones opened the pull request:
### Proposed changes

I got the green light from @dmp42 to add direct download links for CS Engine bits to the docs.
This PR does that. It also normalizes the CS install instructions to be in sync with the non-CS
Engine install instructions.

I also tweaked the CS 1.12 doc structure to match 1.13.

@cpuguy83 I tried to make this very maintainable by using Liquid variables for the package version strings.

Fixes #1026
Fixes #1029

cpuguy83 commented on cs-engine/1.13/index.md:
```diff
@@ -162,30 +223,100 @@ are available:
+| Operating System      | Binary package Links                                                                                                                                                                                                                                                                                                                                               | Source package L ...
+|-----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------- ...
+| RHEL 7.x and CentOS 7 | [docker-engine]({{ rpm-prefix }}/centos/7/Packages/docker-engine-{{ rpm-version}}.el7.centos.x86_64.rpm), [docker-engine-debuginfo]({{ rpm-prefix }}/centos/7/Packages/docker-engine-debuginfo-{{ rpm-version }}.el7.centos.x86_64.rpm), [docker-engine-selinux]({{ rpm-prefix }}/centos/7/Packages/docker-engine-selinux-{{ rpm-version}}1.el7.centos.noarch.rpm) | [docker-engine]( ...
+| RHEL 7.2 (only use if you have problems with `selinux` with the packages above) | [docker-engine]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-{{ rpm-rhel-version }}.el7.centos.x86_64.rpm), [docker-engine-debuginfo]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engine-debuginfo-{{ rpm-rhel-version }}.el7.centos.x86_64.rpm), [docker-engine-selinux]({{ rpm-prefix }}/rhel/7.2/Packages/docker-engin ...
```
These rhel links are broken.

mstanleyjones replied:
They aren't broken anymore with the latest commit. Because the lines here didn't have to change (yay templating) Github won't notice.

cpuguy83 commented on cs-engine/1.13/index.md:
```diff
@@ -162,30 +223,100 @@ are available:
+#### CS Docker Engine 1.13.1
+
+{% comment %} Check on the S3 bucket for packages.docker.com for the versions. {% endcomment %}
+{% assign rpm-version = "1.13.1.cs1-1" %}
```
`rpm-version` and `rpm-rhel-version` should be the same.

joaofnfernandes commented on cs-engine/1.13/index.md:
````diff
@@ -50,109 +63,157 @@ to update its RHEL kernel.
+          $ sudo apt-get install docker-engine=<version>
+          ```
+
+5. Confirm the Docker daemon is running:
````
The snippet below is not rendering properly. I think you're missing a space here.

joaofnfernandes commented on cs-engine/1.13/index.md:
```diff
@@ -50,109 +63,157 @@ to update its RHEL kernel.
-
-2.  Refresh your repository so that curl commands and CA certificates
-are available:
+1.  Refresh your repository:
```
It seems you forgot to include the snippet below.

joaofnfernandes commented on cs-engine/1.12/index.md:
````diff
@@ -1,14 +1,392 @@
+          $ sudo apt-get install docker-engine=<version>
+          ```
+
+5. Confirm the Docker daemon is running:
````
The snippet below is not rendering properly. I think you're missing a space here.

mstanleyjones commented:
Addressed the feedback so far. I was spot-checking the links but I must not have checked the RHEL ones for 1.13. They seem to work with the latest commit.

mstanleyjones commented:
Addressed more feedback!
